.github/ISSUE_TEMPLATE/config.yml

```
@@ -6,3 +6,6 @@ contact_links:
   - name: 🌐 Found a problem with nodejs.org beyond the API reference docs?
     url: https://github.com/nodejs/nodejs.org/issues/new/choose
     about: Please file an issue in the Node.js website repo.
+  - name: 🛠️ Before submitting an issue, please check the support document
+    url: https://github.com/nodejs/node/blob/main/.github/SUPPORT.md
```

In case we ever change the name of the default branch
```suggestion
    url: https://github.com/nodejs/node/blob/HEAD/.github/SUPPORT.md
```